documentation/commands.md

```diff
@@ -222,22 +244,67 @@ Currently, only landings, taxiing, and fix navigation can be aborted.
 
 _Syntax -_ `AAL123 abort`
 
-### Hold
-_Aliases -_ `hold`
+### Altitude
+_Aliases -_ `climb` / `c` / `descend` / `d` / `altitude` / `a`
 
-_Information -_ This command instructs the aircraft to enter a holding
-pattern until further notice. The direction (left/right) may be specified,
-as well as the leg length (in minutes), as well as the fix to hold over.
-But you may also omit those specifications, in which case, the aircraft
-will enter a standard holding pattern over their present position (right
-turns, 1 minute legs). To escape a hold, just set a new heading.
+_Hotkey -_ `up arrow` / `down arrow` (if "Control Method" setting = "Arrow Keys")
 
-_Parameters -_ Specify the direction of turns during the hold with `right`
-or `left`, the leg length, with `[time]min`, and the fix to hold over
-with simply `[fixname]`. Any combination of these arguments provided in
-any order is acceptable, as long as the command `hold` comes first.
+_Information -_ This command tells the specified plane the altitude, in
+hundreds of feet (flight levels), it should travel to. This means that when
+writing altitudes you would drop the last two zeros. For example, 3,000ft =
+"30", 8,300ft = "83", 10,000ft = "100", and FL180 (18,000ft) = "180".
+Airplanes will not descend below 1000 feet (unless locked on ILS).
 
-_Syntax -_ `AAL123 hold [fixname] [left|right] [leg_time]min` or `AAL123 hold`
+Altitude also accepts an `expedite` or `x` argument which can be used as the last item in the command.
+
+_Syntax -_ `AAL123 c [alt]` or `AAL123 c [alt] x`
+
+### Heading
+_Aliases -_ `heading` / `h` / `turn` / `t` / `fh`
+
+_Shortkeys -_ `left arrow` / `right arrow` (if "Control Method" setting = "Arrow Keys")
+
+_Information -_ This command sets the target heading; up (north) is 360,
+right (east) is 090, down (south) is 180, and left (west) is 270. Of course
+you can use any angle in between these as well. If the heading is set
+before takeoff, the aircraft will turn to that heading after takeoff. You
+can force the aircraft to reach the heading by turning left or right by
+inserting `l` or `r` before the new heading, as demonstrated below.
+
+_Syntax -_ `AAL123 fh [hdg]` or `AAL123 (rightarrow) [hdg]` or `AAL123 t r [hdg]`
+
+### Speed
+_Aliases -_ `speed` / `slow` / `sp`
+
+_ShortKey -_ `numpad +` / `numpad -`
+
+_Information -_ This command sets the target speed; aircraft will stay within
+their safe speeds if you tell them to fly faster or slower than they are able
+to. It takes some time to increase and reduce speed. Remember that speed is
+always expressed in knots.
+
+_Syntax -_ `AAL123 - [spd]` or `AAL123 + [spd]`
+
+### Squawk
```

Comment:
This doesn't really belong with the basic control instructions, so let's just make a new `miscellaneous` category. It'll find a logical place as the command list expands in the coming months.